cumulus/client/consensus/aura/src/equivocation_import_queue.rs

```diff
@@ -144,10 +145,10 @@ where
 
 		// check seal and update pre-hash/post-hash
 		{
-			let authorities = self
-				.authorities_tracker
-				.fetch_or_update(&block_params.header, &CompatibilityMode::None)
-				.map_err(|e| format!("Could not fetch authorities: {}", e))?;
+			let authorities = aura_internal::fetch_authorities(self.client.as_ref(), parent_hash)
+				.map_err(|e| {
+				format!("Could not fetch authorities at {:?}: {}", parent_hash, e)
```

Comment:
nit: Formatting looks off

Reply:
Cargo fmt needs it that way https://github.com/paritytech/polkadot-sdk/pull/10072/commits/d186ab0b02b3f74bbfdb93446bc2544c775bf6c4